engines/cge2/vmenu.h

```diff
@@ -76,6 +76,7 @@ class VMenu : public Talk {
 	char *_vmgt;
 	static VMenu *_addr;
 	int _recent;
+	int lastN = 2;
```

Comment:
cge2 is not marked as c++11 - this would break in c++98 compilation mode. You should init the var in the constructor.

Reply:
Is it this particular variable? or all the variables?

Reply:
All variables, and I added a comment regarding another variable with the same issue.
Also the variable name should start with an underscore since it is a class member variable.